### PR TITLE
Project documentation

### PR DESCRIPTION
**Pull request by aaroncohen73**
fixes issue #17 
## Changes in this PR.

- [X] Install and usage documentation
- [X] Developer documentation

More content could be added later, but this should suffice for a start.
## Testing this PR.

1. Run `make html` inside the `docs` folder
2. Still inside the folder, run `firefox build/html/index.html`
### Expected Output.

You should see the documentation

@osuosl/devs

### CODE REVIEW

File: docs/source/devvin.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+WTForms
+'''''''
+
+`WTForms`_ is the library we use to create and handle HTML forms. It provides
+neat features like form validation and `CSRF`_ protection. The Flask-WTF
+allows for easy integration with Flask.
+
+.. _WTForms: http://wtforms.readthedocs.io/en/latest/index.html
+.. _CSRF: https://www.owasp.org/index.php/Cross-Site_Request_Forgery_%28CSRF%29
+
```
Comment (pop) [inline docs/source/devvin.rst:45]: might also want to add a link to https://flask-wtf.readthedocs.io/en/latest/

File: docs/source/devvin.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+body       The body of the webpage
+========== ===============================================
+
+Other static page elements go in the :code:`app/static/` folder.
+
+Style Guidelines
+----------------
+
+This codebase adheres to the `PEP 8`_ standard. Make sure your code
+passes inspection by :code:`flake8` before you submit a pull request!
```
Comment (pop) [inline docs/source/devvin.rst]: Add a codeblock showing an example of running flake8. I know it's stupid but people read code blocks more than they read text.

File: docs/source/devvin.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+.. _PEP 8: https://www.python.org/dev/peps/pep-0008/
+
+Testing
+-------
+
+Writing comprehensive unit tests is good practice. All of the tests for
+timesync-frontend-flask are located in the :code:`tests/` directory, and you can run
+them all at once by running :code:`nosetests` inside the project root directory.
```
Comment (pop) [inline docs/source/devvin.rst]: Add a codeblock showing an example of running tests. I know it's stupid but people read code blocks more than they read text.

**Comment by pop** [comment]
+1

**Comment by Kennric** [comment]
Your usage doc is actually an install doc - usage would be how to log your time, manage users, etc with the app. What is here is good, but we still need actual usage docs, you can add them to this pr, or merge this and create a new issue for them. Either way, please rename 'usage' to 'install' or 'deployment' or 'setup' or something similar.

**Comment by aaroncohen73** [comment]
@Kennric After this PR gets merged I'll create an issue and start working on usage docs

**Pull request merged**